src/Mealz/MealBundle/Tests/Controller/SlotControllerTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mealz\MealBundle\Tests\Controller;
+
+use App\Mealz\MealBundle\Entity\Slot;
+use App\Mealz\UserBundle\DataFixtures\ORM\LoadRoles;
+use App\Mealz\UserBundle\DataFixtures\ORM\LoadUsers;
+
+class SlotControllerTest extends AbstractControllerTestCase
+{
+    /**
+     * @inheritDoc
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->clearAllTables();
+        $this->loadFixtures([
+            new LoadRoles(),
+            new LoadUsers(self::$container->get('security.user_password_encoder.generic')),
+        ]);
+    }
+
+    /**
+     * @test
```

Comment:
We used prefix for unit test method names "public function test..." for all tests. It would be nice if it is consistent.

Reply:
I prefer @test annotation. It makes it easier to quickly identify test methods. Additionally, it keeps the method names short.

But I am open for suggestions or counter arguments. :)

Reply:
I'm fine with both solutions but would vote for consistency.